spec/acceptance/nodesets/sles-11-x64.yml

```diff
@@ -0,0 +1,10 @@
+HOSTS:
```

Comment:
can you please add those spec files to https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot/spec/acceptance/nodesets as well? Did you test if any docker image as well works? We can run docker acceptance tests on travis, but not vagrant